HomeWorks/HW07.Task3/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace HW07.Task3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter your string:");
+            string inputInfo = Console.ReadLine();
+            char[] signs = inputInfo.ToCharArray();
+            string firstNum = string.Empty;
+            int numIndex = 0;
+            int numIndex2 = 0;
+            string secondNum = string.Empty;
+            int mathSignPos = 0;
+            for(int i = 0; i < signs.Length; i++)
+            {
+                    if (signs[i] == '*' || signs[i] == '/' || signs[i] == '+' || signs[i] == '-') mathSignPos = i; 
```

Comment:
also possible to use 
![image](https://user-images.githubusercontent.com/3023032/110580232-4135c180-8179-11eb-9554-2aeeb33b9db9.png)